src/Shared/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingGetPermissionsHasRequest.cs

```diff
@@ -12,7 +12,7 @@ public void Then_The_Url_Is_Correctly_Constructed(long? ukprn, string? hashedId)
     {
         var url = $"permissions";
 
-        var expectedQueryString = $"Ukprn={ukprn}&PublicHashedId={hashedId}";
+        var expectedQueryString = $"Ukprn={ukprn}&accountLegalEntityId={hashedId}";
```

Comment:
Not a huge deal, but the `hashedId` param is confusingly named now that it's an `accountLegalEnttiyId`